components/sankey/elements/links.md

```diff
@@ -21,7 +21,7 @@ To customize the nodes, declare the `<SankeyLinks>` tag as a direct child of `<T
 
 | Parameter | Type and Default&nbsp;Value | Description |
 | --------- | ---- | ----------- |
-| `ColorType` | `SankeyLinksColorType` enum <br /> (`Static`) | The origin of the link color: <ul><li>`Static` - the link color is based on the `Color` property;</li><li>`Source` - the link color is based on the source node color; </li><li>`Target` - the link color is based on the target node color; </li></ul> |
+| `ColorType` | [`SankeyLinksColorType` enum](/blazor-ui/api/telerik.blazor.sankeylinkscolortype) <br /> (`SankeyLinksColorType.Static`) | The origin of the link color: could be based on the `Color` property, on the source node color or on the target node color. |
```

Comment:
In a list of three or more items, the serial comma is a good practice. ([source](https://docs.telerik.com/style-guide/punctuation#series))

```suggestion
| `ColorType` | [`SankeyLinksColorType` enum](/blazor-ui/api/telerik.blazor.sankeylinkscolortype) <br /> (`SankeyLinksColorType.Static`) | The origin of the link color: could be based on the `Color` property, on the source node color, or on the target node color. |
```